schemas/cloud.sql

```diff
@@ -284,7 +285,7 @@ BEGIN
       'summariser'
       FROM TVMUsagePerMonth
       GROUP BY SiteID, CloudComputeServiceID, Month, Year, GlobalUserNameID, VOID,
-          VOGroupID, VORoleID, Status, CloudType, ImageId, CpuCount,
```

Comment:
Should we remove `CloudType` from the rest of the `SummariseVMs` procedure now that we aren't grouping by it?